src/main/java/seedu/duke/command/AddNoteCommand.java

```diff
@@ -2,91 +2,78 @@
 
 import seedu.duke.data.exception.SystemException;
 import seedu.duke.data.notebook.Note;
-import seedu.duke.data.timetable.Event;
 
 import java.util.ArrayList;
 
 import static java.util.stream.Collectors.toList;
-import static seedu.duke.util.PrefixSyntax.PREFIX_DATETIME;
 import static seedu.duke.util.PrefixSyntax.PREFIX_DELIMITER;
-import static seedu.duke.util.PrefixSyntax.PREFIX_RECURRING;
-import static seedu.duke.util.PrefixSyntax.PREFIX_REMIND;
 import static seedu.duke.util.PrefixSyntax.PREFIX_TITLE;
 import static seedu.duke.util.PrefixSyntax.PREFIX_TAG;
 import static seedu.duke.util.PrefixSyntax.PREFIX_PIN;
 
 /**
  * Adds a Note into the Notebook or an Event into the Timetable.
  */
-public class AddCommand extends Command {
+public class AddNoteCommand extends Command {
 
-    public static final String COMMAND_WORD_NOTE = "add-n";
-    public static final String COMMAND_WORD_EVENT = "add-e";
+    public static final String  COMMAND_WORD_NOTE = "add-n";
```

Comment:
Just a minor indentation issue. For all my suggestions, if you are okay with it, feel free to commit directly, using the "commit suggestion"! :)
```suggestion
    public static final String COMMAND_WORD_NOTE = "add-n";
```

Reply:
the name of the variable can be COMMAND_WORD since now add event is split into another class

Reply:
Good call!